build.js

```diff
@@ -31,7 +31,7 @@ var postfix = '\n' + [
   '};'
 ].join('\n');
 
-request('https://raw2.github.com/mkoryak/floatThead/v' + process.argv[2] + '/dist/jquery.floatThead.js',
+request('https://cdnjs.cloudflare.com/ajax/libs/floatthead/' + process.argv[2] + '/jquery.floatThead.js',
```

Comment:
We should still use the version hosted by GitHub as the canonical source.  They have changed their urls recently though, so it should be:

```js
'https://raw.githubusercontent.com/mkoryak/floatThead/v' + process.argv[2] + '/dist/jquery.floatThead.js
```